core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -194,7 +194,7 @@
 
   public static final Splitter ACCEPT_ENCODING_SPLITTER = Splitter.on(',').trimResults();
 
-  private static final String IMPLEMENTATION_VERSION = getImplementationVersion();
+  private static final String IMPLEMENTATION_VERSION = "1.9.0-SNAPSHOT"; // CURRENT_GRPC_VERSION
```

Comment:
according to `Package`, the version needs to be numerics and dots, no snapshot version.

Reply:
What is the implication?